files/definitions/2.2.0-github1

```diff
@@ -0,0 +1,3 @@
+install_package "yaml-0.1.6" "http://pyyaml.org/download/libyaml/yaml-0.1.6.tar.gz#5fe00cda18ca5daeb43762b80c38e06e" --if needs_yaml
+install_package "openssl-1.0.1j" "https://www.openssl.org/source/openssl-1.0.1j.tar.gz#f7175c9cd3c39bb1907ac8bba9df8ed3" mac_openssl --if has_broken_mac_openssl
+install_package "ruby-2.2.0-github1" "https://s3.amazonaws.com/boxen-downloads/ruby-build/ruby-2.2.0-github1.tar.gz#2e538443e5ce80da1d922153d1422439" ldflags_dirs autoconf standard verify_openssl
```

Comment:
our 2.2 branch is very much in flux and we're not using it anywhere so I don't think we should ship binaries for it